pos_self_service_base/readme/cups

```diff
@@ -0,0 +1,52 @@
+upstream cups-reverse-proxy {
```

Comment:
Indeed, it should be in [`pos_self_service_print_zpl`](https://github.com/OCA/pos/pull/656) (or simply removed since it's in `pos_self_service_print_zpl/readme/CONFIGURE.rst`)